### ISSUE TITLE
"could not open compilation database for /path/to/Foo.swift"

### ISSUE THREAD

**Issue by dabrahams** [emacs-lsp/lsp-sourcekit#3]
I can only get lsp-sourcekit to give me  the following in `*Messages*` buffer:

```
could not find manifest, or not a SwiftPM package: /path/to/current/directory
could not open compilation database for /path/to/current/directory/Foo.swift [3 times]
```

Do I need to have a SwiftPM package for my project in order to do anything?

**Comment by danielmartin** [comment]
Yes, for now sourcekit-lsp requires a Swift PM package that provides the necessary compile flags. The second warning can be ignored for Swift projects. It only applies to projects that provide compilation flags using a compilation database (https://clang.llvm.org/docs/JSONCompilationDatabase.html).

**Comment by charshep** [comment]
I'm trying to get this working with Xcode 11.4 and I'm seeing the same errors. Is it still the case that lsp-sourcekit only works for projects that have a Swift PM package? If so, perhaps the quickstart guide should indicate that.
